test/tcregexp.pas

```diff
@@ -273,49 +274,56 @@   TRegExTest = record
     substitutionText: '';
     expectedResult: 'Test: hel'#$d#$a'lo ;';
     matchStart: 1
+    ),
+    (
+    expression: '(?:\w+)=\w+;(\w+)=\w+;(?:\w+)=\w+;(\w+)=\w+;';
+    inputText: 'skip1=11;needed1=22;skip2=33;needed2=44;';
+    substitutionText: '$1 $2';
+    expectedResult: 'needed1 needed2';
+    matchStart: 0
     )
   );
 
 procedure TTestRegexpr.IsFalse(ArrorMessage: string; AConditionToCheck: boolean);
 begin
-  IsTrue(ArrorMessage, not AConditionToCheck)
+  IsTrue(ArrorMessage, not AConditionToCheck)
 end;
 
 procedure TTestRegexpr.IsTrue(ArrorMessage: string; AConditionToCheck: boolean);
 begin
-  {$IFDEF FPC}
-  AssertTrue(ArrorMessage, AConditionToCheck);
-  {$ELSE}
-  CheckTrue(AConditionToCheck, ArrorMessage)
-  {$ENDIF}
+  {$IFDEF FPC}
+  AssertTrue(ArrorMessage, AConditionToCheck);
+  {$ELSE}
+  CheckTrue(AConditionToCheck, ArrorMessage)
+  {$ENDIF}
 end;
 
 procedure TTestRegexpr.IsNotNull(ArrorMessage: string; AObjectToCheck: TObject);
 begin
-  {$IFDEF FPC}
-  AssertNotNull(ArrorMessage, AObjectToCheck);
-  {$ELSE}
-  CheckNotNull(AObjectToCheck, ArrorMessage)
-  {$ENDIF}
+  {$IFDEF FPC}
+  AssertNotNull(ArrorMessage, AObjectToCheck);
+  {$ELSE}
+  CheckNotNull(AObjectToCheck, ArrorMessage)
+  {$ENDIF}
 end;
 
 procedure TTestRegexpr.AreEqual(ArrorMessage: string; s1,s2: string);
 begin
-  {$IFDEF FPC}
-  AssertEquals(ArrorMessage, s1,s2);
-  {$ELSE}
-  CheckEquals(s1,s2, ArrorMessage)
-  {$ENDIF}
-end;
+  {$IFDEF FPC}
```

Comment:
not clear - what was changed?

Reply:
It is some EOL changes made by Lazarus, or maybe trailing spaces deleted.